.buildkite/pipeline.yml

```diff
@@ -23,7 +23,11 @@ steps:
      - ./binaries/*
      - ./deb-packages/*
      - ./rpm-packages/*
-   branches: !master
+   branches: "!master"
+ - commands:
+   - nix-build -A test-binaries --no-out-link
+   label: test binaries
+   branches: "!master"
```

Comment:
why not master?

Reply:
Autorelease builds everything, so why build this stuff twice?

Reply:
And if we merged PR we are sure that binaries work